.travis.yml

```diff
@@ -4,7 +4,14 @@ before_install: set -e
 after_script: set +e
 install: curl -#fSLo ~/rebar3 https://s3.amazonaws.com/rebar3/rebar3 && chmod +x ~/rebar3
 after_install: ~/rebar3 version
-script: ~/rebar3 eunit
+script:
```

Comment:
what about having it as a pre script in rebar ? Or maybe using a makefile so we can also fetch it easily by hand